### DISCUSSION TITLE
Parameters outside of bounds

### DISCUSSION THREAD

**Discussion by tomko-neuron** [BlueBrain/BluePyOpt#485] [Q&A]
Hi, I am using BluePyOpt to optimize the parameters of a conductance-based model. However, after the optimization is finished (120 generations, 60 offspring), some parameters of the best model are out of bounds, for example Ra.basal = 1.35e-05, while the bounds are [100,400]. What could be the reason for this? I would assume that the parameters cannot be generated outside the bounds.

**Comment by AurelienJaquier** [comment]
Hi @tomko-neuron !
Could you tell me what is the version of BluePyOpt you are using? Also which optimiser did you use? `bluepyopt.deapext.optimisations.IBEADEAPOptimisation` or `bluepyopt.deapext.optimisationsCMA.DEAPOptimisationCMA`? If the latter, did you use `selector_name="single_objective"` or `selector_name="multi_objective"`?

**Reply by tomko-neuron** [reply]
Hi @AurelienJaquier, I am using BluePyOpt version 1.14.7. and `bluepyopt.deapext.optimisations.IBEADEAPOptimisation` and `selector_name="IBEA"`.

**Reply by AurelienJaquier** [reply]
Indeed it is strange that the parameter output is out of bounds. Are you sure that you are not mixing the parameters? That can happen if we are not careful.
If e.g. another parameters has a value that would fall into the bounds of Ra.basal, this might be the case.
If you are positive that BluePyOpt does not respect the bounds, could you maybe share your script and data, so that I can try to reproduce and isolate the source of the error?
